recipe/meta.yaml

```diff
@@ -12,9 +13,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: {{ build }}
+  skip: True  # [py<37]
   script: {{ PYTHON }} -m pip install . --no-deps -vv
-  noarch: python
   entry_points:
     - google-oauthlib-tool = google_auth_oauthlib.tool.__main__:main
 
```

Comment:
Could you please rename requirements/build to requirements/host? And add `setuptools` and `wheel` to host?
See https://github.com/googleapis/google-auth-library-python-oauthlib/blob/830651eff490c47aeb3ba0b6b27afa9f92ddccdf/setup.cfg#L18 and https://github.com/googleapis/google-auth-library-python-oauthlib/blob/830651eff490c47aeb3ba0b6b27afa9f92ddccdf/setup.py#L18
Use a plain `python` without pinning in `host` and `run` for `arch-specific` python packages. You already skipped `py<37`.